### PR TITLE
PR: Input Component

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added different types of Input Components
  - Text
  - Password
  - Email
  - Numeric
- Added Disabled
- Added valid and invalid
- Added Radio buttons and Checkbox

Kindly review and let me know the changes
TIA!

https://user-images.githubusercontent.com/91378157/154815314-94c1df6d-c551-4d53-b3fc-3dfa24828e48.mp4

### CODE REVIEW

File: Components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    border: 2px solid var(--primary-color);
+}
+
+
+.active-field {
+    border: 2px solid var(--primary-color);
+}
+
+.disabled {
+    color: #c5c5c5;
```
Comment (krgarima) [inline Components/input/input.css]: Use CSS variables

**Review by krgarima — COMMENTED** [review]
Site looks very beautiful. Good to go.

File: Components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+.label-required-field::after {
+    content: "*";
+    color: var(--primary-color);
+}
+
+.field {
+    width: 100%;
+    display: block;
+    padding: .8rem;
+    border: 1px solid #d4d5d9;
```
Comment (vikrantkode) [inline Components/input/input.css]: use variables to define colors

File: Components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    box-sizing: border-box;
+    font-size: 1.4rem;
+}
+
+.field-hover {
+    width: 100%;
+    display: block;
+    padding: .8rem;
+    border: 1px solid #d4d5d9;
+    border-radius: 0.5rem;
```
Comment (vikrantkode) [inline Components/input/input.css]: same here

File: Components/input/input.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+.warning-lbl {
+    color: #ff1158;
+}
+
+.valid {
+    border-color: #a4ff11;
+}
+
+.valid-lbl {
+    color: #36e709;
```
Comment (vikrantkode) [inline Components/input/input.css]: same here

**Review by vikrantkode — COMMENTED** [review]
suggested few changes. Rest good to go

File: Components/input/input.css (hunk 1 above)
Comment (Jayesh3636) [reply Components/input/input.css]: Thanks! Changes have been made! any other review?

File: Components/input/input.css (hunk 2 above)
Comment (Jayesh3636) [reply Components/input/input.css]: Thanks for reviewing! required changes have been made!

File: Components/input/input.css (hunk 4 above)
Comment (Jayesh3636) [reply Components/input/input.css]: required changes have been made!

**Comment by Jayesh3636** [comment]
> Site looks very beautiful. Good to go.



> Site looks very beautiful. Good to go.

thanks!

**Pull request merged**